tests/test_drs.py

```diff
@@ -81,6 +103,163 @@ def test_drs_get_no_default(client, user):
     settings["config"]["INDEX"]["driver"].config["ADD_PREFIX_ALIAS"] = True
 
 
+def test_timestamps(client, user):
+    data = get_doc()
+    create_obj_resp = client.post("/index/", json=data, headers=user)
+    assert create_obj_resp.status_code == 200
+    obj_did = create_obj_resp.json["did"]
+    obj_resp = client.get(f"/ga4gh/drs/v1/objects/{obj_did}")
+    assert obj_resp.status_code == 200
+    assert data["created_time"] == obj_resp.json["created_time"]
+    assert data["updated_time"] == obj_resp.json["updated_time"]
+
+
+def test_changing_timestamps(client, user):
+    data = get_doc()
+    create_obj_resp = client.post("/index/", json=data, headers=user)
+    assert create_obj_resp.status_code == 200
+    obj_did = create_obj_resp.json["did"]
+    obj_rev = create_obj_resp.json["rev"]
+    update_json = {
+        "created_time": "2023-03-15T17:02:54",
+        "updated_time": "2023-03-30T17:02:54",
+    }
+    update_obj_resp = client.put(
+        f"/index/{obj_did}?rev={obj_rev}", json=update_json, headers=user
+    )
+    assert update_obj_resp.status_code == 200
+    update_obj_did = update_obj_resp.json["did"]
+    obj_resp = client.get(f"/ga4gh/drs/v1/objects/{update_obj_did}")
+    assert obj_resp.status_code == 200
+    assert update_json["created_time"] == obj_resp.json["created_time"]
+    assert update_json["updated_time"] == obj_resp.json["updated_time"]
+
+
+def test_changing_timestamps_updated_not_before_created(client, user):
```

Comment:
this test and the 3 following tests are not specific to DRS, preferably they should be in the generic indexd tests

Reply:
Moved these tests to test_client